package.json

```diff
@@ -35,5 +21,21 @@
     "fastclick": "1.0.6",
     "bootstrap": "3.3.6",
     "slipjs": "pornel/slip#0626afc"
```

Comment:
Not sure, but whenever npm install is run, slipjs is updated. That generally not happening with other libraries. Is it because we specify a commit instead of a version? Could we address this?